frontend/src/features/splReports/components/ReportControls.tsx

```diff
@@ -146,7 +147,7 @@ const ReportControls: React.FunctionComponent<IReportControlsProps> = ({
                 <Button
                   className="h-75 mr-auto"
                   type="submit"
-                  disabled={!dirty || !values.id}
+                  disabled={(!dirty && originalReport?.to === currentReport?.to) || !values.id}
```

Comment:
Since the form gets re-initialized when the data changes during refresh, we lose the fact that the to date has changed. This ensures that the save button is enabled consistently in this case.